tg_bot/modules/chatlist.py

```diff
@@ -0,0 +1,34 @@
+from io import BytesIO
+from typing import Optional, List
+from telegram import TelegramError, Chat, Message
+from telegram import Update, Bot
+from telegram.ext import MessageHandler, Filters, CommandHandler
+from telegram.ext.dispatcher import run_async
+
+import tg_bot.modules.sql.users_sql as sql
+from tg_bot import dispatcher, OWNER_ID, LOGGER
+from tg_bot.modules.helper_funcs.filters import CustomFilters
+
+USERS_GROUP=4
+
+@run_async
+def chats(bot: Bot, update: Update):
+    chats = sql.get_all_chats() or []
+
+    chatfile = 'List of chats.\n'
+    for chat in chats:
+        chatfile += "[x] {} - {}\n".format(chat.chat_name, chat.chat_id)
```

Comment:
whats with this formatting, why is there [x]? Its there to mark the user is gbanned in gbanlist, but why here